lib/library/ModernModuleLibraryPlugin.js

```diff
@@ -107,7 +108,7 @@ class ModernModuleLibraryPlugin extends AbstractLibraryPlugin {
 
 				for (const reexportInfo of exp.orderedExports) {
 					if (
-						!reexportInfo.provided &&
+						reexportInfo.provided === false &&
```

Comment:
When `provided` equals null means this module has unknown exports, we should make sure they have export as well